webapp/src/App.js

```diff
@@ -172,17 +171,6 @@ function AppRoutes() {
             LedgersOverview
           )}
         />
-        <Route
```

Comment:
The idea was to put these on one page, not to have a separate page for each ledger (since we are starting with just one).